lib/dishes.rb

```diff
@@ -0,0 +1,25 @@
+class Dishes
+  attr_reader :name, :price, :available, :list_available_dishes
+
+  def initialize(name, price, available = true)
+    $name = name
```

Comment:
I'm not familiar with using a global variable in this context, what's the plan for it?